lib/IDE/CodeCompletion.cpp

```diff
@@ -323,6 +309,63 @@ void CodeCompletionString::dump() const {
   }
 }
 
+ContextFreeCodeCompletionResult *ContextFreeCodeCompletionResult::create(
```

Comment:
Would giving these more descriptive names make sense? I haven't checked all the callsites, but it seems like maybe we could have `createDeclResult`, `createKeywordResult`, etc. Could make it easier to understand which (if any) should be used (and where).

Reply:
Ah, that's a good idea 👍